infra/src/main/resources/db/migration/prod/V61__add_split_table.sql

```diff
@@ -0,0 +1,19 @@
+create type publication.split_push_state as enum ('PENDING', 'IN_PROGRESS', 'DONE', 'FAILED');
+
+create table publication.split
+(
+    id                          int                             primary key generated always as identity,
+    source_location_track_id    int                             not null,
+    state                       publication.split_push_state    not null default 'PENDING',
+    error_cause                 text                            null default null,
```

Comment:
Meillä on muualla tainnut olla patternina, että ollaan pidetty virheet omassa taulussaan. Suurimmalla osalla tämä (toivottavasti) tulee kumminkin jäämään nulliksi. Mulle itselleni kyllä sopii sekin, että jätetään tämä näin sillä se yksinkertaistaa kantarakennetta kivasti, mutta mitä muut on mieltä asiasta?

Reply:
Koitin miettiä että miksi ne olisi eri taulussa ja ainoa mikä tulee mieleen on että integraatioissa voi olla 1-n suhde, eli monta virhettä yhdellä pushilla. Liekö ratkolla noin? Tässä kuvittelisin että virheitä voi olla vain 1, joten tuolla ei ehkä väliä.

Eli minusta ihan ok näin ellei joku keksi jotain syytä miksi ne pitäisi olla omassa taulussaan.